### PR TITLE
add: `ventoy-bin`

### PR DESCRIPTION
**Pull request by Xdavius**

### CODE REVIEW

**Comment by Xdavius** [comment]
PacReq: ventoy #2278

File: packages/ventoy-bin/ventoy-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+arch=('amd64')
+url="http://www.ventoy.net"
+license=('GPL-3.0-or-later')
+depends=('bash' 'util-linux' 'xz-utils' 'dosfstools')
+optdepends=('libgtk-3-dev: GTK3 GUI'
+  'qtbase-5-dev: Qt5 GUI'
+  'polkit: run GUI from application menu')
+provides=("${pkgname%-bin}")
+conflicts=("${pkgname%-bin}")
+install="${pkgname%-bin}.install"
```
Comment (vigress8) [inline packages/ventoy-bin/ventoy-bin.pacscript:15]:
```suggestion
```

File: packages/ventoy-bin/ventoy-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,90 @@
+pkgname=ventoy-bin
```
Comment (vigress8) [inline packages/ventoy-bin/ventoy-bin.pacscript:1]:
```suggestion
pkgname=ventoy-bin
gives=ventoy
```
Replace occurrences of `${pkgname%-bin}` with `gives`

File: packages/ventoy-bin/ventoy-bin.pacscript [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  'SKIP'
+  'SKIP'
+  'SKIP'
+  'SKIP'
+  'SKIP'
+  'SKIP')
+
+prepare() {
+  cd "${_archive}"
+  export CARCH="x86_64"
```
Comment (vigress8) [inline packages/ventoy-bin/ventoy-bin.pacscript:37]: Replace occurrences of `CARCH` with `x86_64`. There's only one `arch` anyway so it's redundant
```suggestion
```

File: packages/ventoy-bin/ventoy-bin.pacscript [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  # Clean up unused binaries
+  # Preserving mkexfatfs and mount.exfat-fuse because exfatprogs is incompatible
+  for binary in xzcat hexdump; do
+    rm -fv tool/"${CARCH}"/"${binary}"
+  done
+}
+
+package() {
+  cd "${_archive}"
+  export CARCH="x86_64"
```
Comment (vigress8) [inline packages/ventoy-bin/ventoy-bin.pacscript:67]:
```suggestion
```

File: packages/ventoy-bin/ventoy-bin.pacscript [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  'SKIP'
+  'SKIP'
+  'SKIP'
+  'SKIP')
+
+prepare() {
+  cd "${_archive}"
+  export CARCH="x86_64"
+  # Decompress tools
+  pushd tool/"${CARCH}" || exit
```
Comment (vigress8) [inline packages/ventoy-bin/ventoy-bin.pacscript:39]:
```suggestion
  pushd tool/"${CARCH}"
```
Redundant

File: packages/ventoy-bin/ventoy-bin.pacscript [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+  for file in *.xz; do
+    xzcat "${file}" >"${file%.xz}"
+    chmod +x "${file%.xz}"
+  done
+
+  # Cleanup .xz crap
+  rm -fv ./*.xz
+  popd || exit
```
Comment (vigress8) [inline packages/ventoy-bin/ventoy-bin.pacscript:47]:
```suggestion
  popd
```

File: packages/ventoy-bin/ventoy-bin.pacscript [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+  'SKIP'
+  'SKIP'
+  'SKIP')
+
+prepare() {
+  cd "${_archive}"
+  export CARCH="x86_64"
+  # Decompress tools
+  pushd tool/"${CARCH}" || exit
+  for file in *.xz; do
```
Comment (vigress8) [inline packages/ventoy-bin/ventoy-bin.pacscript:40]:
```suggestion
  for file in ./*.xz; do
```

**Review by vigress8 — CHANGES_REQUESTED** [review]

File: packages/ventoy-bin/ventoy-bin.pacscript (hunk 3 above)
Comment (Xdavius) [reply packages/ventoy-bin/ventoy-bin.pacscript:37]: > Replace occurrences of `CARCH` with `x86_64`. There's only one `arch` anyway so it's redundant

ok

File: packages/ventoy-bin/ventoy-bin.pacscript (hunk 5 above)
Comment (Xdavius) [reply packages/ventoy-bin/ventoy-bin.pacscript:39]: This was suggested by shellcheck ...

File: packages/ventoy-bin/ventoy-bin.pacscript (hunk 2 above)
Comment (vigress8) [reply packages/ventoy-bin/ventoy-bin.pacscript:1]: Still haven't done this

**Review by Elsie19 — CHANGES_REQUESTED** [review]
Where is `.SRCINFO`?

**Comment by vigress8** [comment]
Generate it by running
```
./scripts/srcinfo.sh packages/ventoy-bin/ventoy-bin.pacscript
```

**Comment by Xdavius** [comment]
> Generate it by running
> 
> ```
> ./scripts/srcinfo.sh packages/ventoy-bin/ventoy-bin.pacscript
> ```

I keep it in mind

**Review by oklopfer — APPROVED** [review]

**Pull request merged**